tests/js-tests/src/SysTest/SysTest.js

```diff
@@ -65,6 +65,23 @@ var SysTestBase = BaseTestLayer.extend({
 
 //------------------------------------------------------------------
 //
+// setClearColorTest
+//
+//------------------------------------------------------------------
+var setClearColorTest = SysTestBase.extend({
+    _title:"Set clearColor to red with alpha=255",
+    ctor:function()
+    {
+        this._super();
+
+        var clearColor = new cc.color(255, 0, 0, 255);
+        director.setClearColor(clearColor);
```

Comment:
Suggest to set the clear color in `onEnter` and reset the clear color in `onExit` to avoid affect other test cases